test/python/compiler/test_transpiler.py

```diff
@@ -860,6 +860,25 @@ def test_transpile_subset_of_calibrated_gates(self):
         transpiled_circ = transpile(circ, FakeAlmaden())
         self.assertEqual(set(transpiled_circ.count_ops().keys()), {'u2', 'mycustom', 'h'})
 
+    def test_parameterized_calibrations_transpile(self):
+        """Check that gates can be matched to their calibrations before and after parameter
+        assignment."""
+        tau = Parameter('tau')
+        circ = QuantumCircuit(3, 3)
+        circ.append(Gate('rxt', 1, [2*3.14*tau]), [0])
+
+        def q0_rxt(tau):
+            with pulse.build() as q0_rxt:
+                pulse.play(pulse.library.Gaussian(20, 0.4*tau, 3.0), pulse.DriveChannel(0))
+            return q0_rxt
+        circ.add_calibration('rxt', [0], q0_rxt(tau), [2*3.14*tau])
+
+        transpiled_circ = transpile(circ, FakeAlmaden())
+        self.assertEqual(set(transpiled_circ.count_ops().keys()), {'rxt'})
```

Comment:
Could also test that the calibration has been unaffected by the transpilation. Alternatively, if you're just checking for whether `transpile` raises you can use the
```try:
    transpile(...)
except ExceptionType:
    self.fail(....)
```
method.

Reply:
Hi @ewinston, you are mentioning the scope of this PR https://github.com/Qiskit/qiskit-terra/pull/5049/files where that test has already been added

Reply:
although that one is not parameterized